app/controllers/application_controller.rb

```diff
@@ -14,7 +14,8 @@ def tour_params
   end
 
   def get_tours
-    @pagy, @tours = pagy Tour.by_id(get_tour_ids).by_title(params[:title]),
+    @pagy, @tours = pagy Tour.by_title(params[:title])
+                             .includes(:tour_schedules),
```

Comment:
đăc điểm của includes

Reply:
includes sử dụng select in() để thực hiện truy vấn
includes sinh lỗi khi dùng truy vấn theo bảng `tour_schedule` do select in() không load trước dữ liệu từ bảng này.